README.adoc

```diff
@@ -1,53 +1,54 @@
-# prettier-config-keep
+# prettier-config
 
-https://prettier.io/[Prettier] code formatter configuration shared across Keep
-Network projects.
+https://prettier.io/[Prettier] code formatter configuration shared across Thesis projects.
 
 ## Usage
 
 ### Setting up a project
 
 1. Install dependencies:
+
 +
 ```sh
-  npm install --save-dev github:keep-network/prettier-config-keep prettier
+  yarn add --Dev github:thesis/prettier-config prettier
 ```
 
+
 2. Configure prettier:
 +
 The easiest way to configure prettier to use this configuration is to define 
 `prettier` property in the project's `package.json`:
 +
 ```json
-"prettier": "@keep-network/prettier-config-keep"
+"prettier": "@thesis/prettier-config"
 ```
 +
 For more sophisticated configuration instead of adding this entry add a configuration
 file (see: <<Plugins>> section).
 
-3. Define `scripts` property in `package.json`:
+3. Define `scripts` property in `package.json`, e.g.:
```

Comment:
Thoughts on adding a note here that for JavaScript/TypeScript formatting in particular, Thesis projects should use the prettier-eslint integration in `@thesis/eslint-config`?

Reply:
Sure thing 5e70902